addons/website_payment_paypal/__manifest__.py

```diff
@@ -8,9 +8,6 @@
     'sequence': 365,
     'summary': 'Website - Payment Paypal',
     'depends': ['website_payment', 'payment_paypal'],
-    'data': [
-        'views/res_config_settings_views.xml'
-    ],
```

Comment:
The module `website_payment_paypal` was only useful for this settings block. You can therefore remove the module :)